### PR TITLE
docs: add more usage details

### PR DESCRIPTION
**Pull request by jsstevenson**
Added some more information regarding database construction in response to a question on the GA4GH slack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,28 @@
...
+
+Create a database instance and pass it to the `QueryHandler` constructor:
+
+```
+>>> from gene.query import QueryHandler
+>>> from gene.database import create_db
+>>> q = QueryHandler(create_db())
+```
+
+Call the `normalizer()` method with a gene term. If available, a rich description of the normalized concept is returned.
```
Comment (korikuzma) [inline README.md:17]:
```suggestion
Call the `normalize()` method with a gene term. If available, a rich description of the normalized concept is returned.
```

File: docs/source/usage.rst [hunk 2]
```diff
@@ -38,6 +38,28 @@ Each search mode can be accessed directly within Python using the :ref:`query AP
...
+
+Users hoping for a more explicit connection declaration may instead call a database class directly, e.g.:
+
+.. code-block:: python
+
+    from gene.database.postgresql import PostgresDatabase
+    from gene.query import QueryHandler
+    pg_db = PostgresDatabase(
+        user="postgres",
+        password="hunter2",
```
Comment (korikuzma) [inline docs/source/usage.rst:56]: can we do `matthew_cannon2` as the password

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jsstevenson) [reply README.md:17]: ack, went robot mode

**Review by korikuzma — APPROVED** [review]
💯

**Pull request merged**